Data/models/Model.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Model8.xcdatamodel</string>
+	<string>Model9.xcdatamodel</string>
```

Comment:
Instead of creating a new model9, just make this change on model8. Model8 is new too, will break us dev's instances, but that isn't a big issue.